docs/release-notes.md

```diff
@@ -12,12 +12,15 @@ nav_order: 9
 
 ### Features
 
+- Stabilize OpenShift spec 4.13.0, targeting Ignition spec 3.2.0
+- Add OpenShift spec 4.14.0-experimental, targeting Ignition spec
+  3.4.0
 - Allow enabling discard passthrough on LUKS devices _(fcos 1.5.0-exp,
-  flatcar 1.1.0-exp, openshift 4.13.0-exp)_
+  flatcar 1.1.0-exp, openshift 4.14.0-exp)_
 - Allow specifying arbitrary LUKS open options _(fcos 1.5.0-exp,
-  flatcar 1.1.0-exp, openshift 4.13.0-exp)_
-- Allow specifying user password hash _(openshift 4.13.0-exp)_
-- Support offline Tang provisioning via pre-shared advertisement _(fcos 1.5.0-exp, openshift 4.13.0-exp)_
+  flatcar 1.1.0-exp, openshift 4.14.0-exp)_
+- Allow specifying user password hash _(openshift 4.13.0)_
```

Comment:
We need to also say that it's going into 4.14.0-exp, spelled _(openshift 4.13.0+)_.